machine/v1beta1/types_machinehealthcheck.go

```diff
@@ -6,6 +6,12 @@ import (
 	"k8s.io/apimachinery/pkg/util/intstr"
 )
 
+const (
+	// PausedAnnotation is an annotation that can be applied to MachineHealthCheck objects to prevent the MHC controller
+	// from processing it.
+	MHCPausedAnnotation = "cluster.x-k8s.io/paused"
```

Comment:
is the machine health check controller an upstream controller or downstream one?  I thought it was a downstream controller that watched `MachineHealthCheck.machine.openshift.io`.  If it is a downstream controller and a downstream API, this seems like a good choice for either 
1. a spec field
2. handling by simply deleting the machinehealthcheck

Reply:
The machinehealthcheck is a fork of the upstream controller which is currently feature paired. We do have our own CRDs but they match the upstream identically at the moment.

The paused feature is something we picked from their implementation and we decided, to avoid potential future discrepancies between OCP and HyperShift, to keep the same annotation.

This was shipped in 4.9 and so I don't think we want to make a change to this now, though I appreciate based on our prior conversations about API compatibility that we should have made this a first class field when we backported it